### PR TITLE
Drop "to Clipboard" from ZAP copy menu items

### PR DESCRIPTION
**Pull request by kingthorin**
Part of zaproxy/zaproxy#8179

### CODE REVIEW

File: zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties [hunk 1]
```diff
@@ -556,7 +556,7 @@ ascan.progress.chart.4xx = 4xx
 ascan.progress.chart.5xx = 5xx
 ascan.progress.chart.responses = Responses / second
 ascan.progress.chart.time = Time 
-ascan.progress.copyclipboard.button.label = Copy to Clipboard
+ascan.progress.copyclipboard.button.label = Copy
 ascan.progress.copyclipboard.button.tooltip = Copies the contents of the table to the clipboard
 ascan.progress.copyclipboard.error = Failed to copy the contents to the clipboard.
```
Comment (thc202) [inline zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties:561]: Should drop from there too?

Comment (kingthorin) [reply zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties:561]: Sure. I kinda thought these were more purposefully explanatory but it doesn't really add much, I think clipboard is still a safe assumption here.

**Comment by kingthorin** [comment]
Tweaked

**Review by ricekot — APPROVED** [review]
Thanks!

**Review by psiinon — APPROVED** [review]

**Comment by thc202** [comment]
There are two more occurrences of clipboard that could be dropped (IMO).

**Comment by kingthorin** [comment]
I don't see any occurrences of "to Clipboard" in *.html or Messages.properrties. There's one in *.java but it seems reasonable as part of a logged message for scan progress.

**Comment by thc202** [comment]
e.g. https://github.com/zaproxy/zaproxy/blob/6609be13464a9bd611d0dc2d54893b0310c84cf1/zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties#L1302

**Comment by kingthorin** [comment]
Okay I was being too literal, I was only looking for "to Clipboard"

**Comment by kingthorin** [comment]
Now includes that change.

**Comment by thc202** [comment]
Only one was changed, on purpose?

**Comment by kingthorin** [comment]
Because that's what was discussed. The other was a log message that seemed reasonable to me sir another I hadn't found.

**Comment by thc202** [comment]
The two occurrences I was referring to were in the resources messages, other one being: https://github.com/zaproxy/zaproxy/blob/6609be13464a9bd611d0dc2d54893b0310c84cf1/zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties#L2283

**Comment by kingthorin** [comment]
I probably skipped it because it was paste. Thanks, tweak coming.

**Comment by kingthorin** [comment]
Tweaked.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

@ricekot or @psiinon do you want to check again?

**Comment by ricekot** [comment]
Still looks good to me, thanks!

**Pull request merged**